<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AoGetMainWindow</class>
 <widget class="QMainWindow" name="AoGetMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1002</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AOGet</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2" stretch="35,65">
    <property name="spacing">
     <number>10</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="toolTip">
       <string>A job is a collection of links</string>
      </property>
      <property name="title">
       <string>Jobs</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout" rowstretch="1,0,0,0,0" columnstretch="1,0">
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>15</number>
       </property>
       <property name="horizontalSpacing">
        <number>15</number>
       </property>
       <property name="verticalSpacing">
        <number>5</number>
       </property>
       <item row="0" column="0" rowspan="5">
        <widget class="QTableWidget" name="tblJobs">
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="btnCreateNew">
         <property name="toolTip">
          <string>Download a new set of files from a webpage</string>
         </property>
         <property name="text">
          <string>Create new...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="btnImport">
         <property name="toolTip">
          <string>Import a previously exported job</string>
         </property>
         <property name="text">
          <string>Import...</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="btnExport">
         <property name="toolTip">
          <string>Export selected job</string>
         </property>
         <property name="text">
          <string>Export...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Files in current job</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>15</number>
       </property>
       <item>
        <widget class="QTableWidget" name="tblFiles"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1002</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuApplication">
    <property name="title">
     <string>Application</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="actionOpen_GitHub_page"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuJobs">
    <property name="title">
     <string>Downloads</string>
    </property>
    <addaction name="actionPause_all"/>
    <addaction name="actionResume_all"/>
    <addaction name="separator"/>
    <addaction name="actionSnail_mode"/>
    <addaction name="actionSet_global_bandwidth_limit"/>
    <addaction name="actionScheduled_download"/>
    <addaction name="separator"/>
    <addaction name="actionLoad_set"/>
    <addaction name="actionSave_this_set_as"/>
    <addaction name="separator"/>
    <addaction name="actionExit_2"/>
   </widget>
   <widget class="QMenu" name="menuSearch">
    <property name="title">
     <string>Search</string>
    </property>
    <addaction name="actionSearch_for"/>
   </widget>
   <addaction name="menuJobs"/>
   <addaction name="menuApplication"/>
   <addaction name="menuSearch"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionOpen_GitHub_page">
   <property name="text">
    <string>Open GitHub Page</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Donate</string>
   </property>
  </action>
  <action name="actionPause_all">
   <property name="text">
    <string>Pause all</string>
   </property>
  </action>
  <action name="actionResume_all">
   <property name="text">
    <string>Resume all</string>
   </property>
  </action>
  <action name="actionSet_global_bandwidth_limit">
   <property name="text">
    <string>Set global bandwidth limit</string>
   </property>
  </action>
  <action name="actionSnail_mode">
   <property name="text">
    <string>Snail mode</string>
   </property>
   <property name="toolTip">
    <string>Keep downloads going but severely limit bandwidth (see Settings)</string>
   </property>
  </action>
  <action name="actionSave_this_set_as">
   <property name="text">
    <string>Save current jobset as...</string>
   </property>
  </action>
  <action name="actionLoad_set">
   <property name="text">
    <string>Load jobset...</string>
   </property>
  </action>
  <action name="actionScheduled_download">
   <property name="text">
    <string>Scheduled download...</string>
   </property>
  </action>
  <action name="actionExit_2">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSearch_for">
   <property name="text">
    <string>Search for...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
